### PR TITLE
Refactored components that used deprecated componentWillRecieveProps() lifecycle handler to no longer use it.

### PR DESCRIPTION
**Pull request by samuel-zuk**
addresses [#249](https://github.com/engaging-computing/MYR/issues/249)

### CODE REVIEW

File: src/components/structural/header/Header.js [hunk 1]
```diff
@@ -292,35 +272,31 @@ class Header extends Component {
...
     * @summary - This function produces the form for inputting the scene's name and description
     */
     sceneName = () => {
-        let text = "";
-        if (this.state.sceneName === null) {
-            text = this.props.scene.name;
-        } else {
-            text = this.state.sceneName;
-        }
+        let nameText = this.props.scene.name;
```
Comment (jasondkiesling) [inline src/components/structural/header/Header.js]: Can we name these sceneName and sceneDesc? It's not immediately clear on what these variables when you first read them.

**Comment by jasondkiesling** [comment]
Please see the directions in #myr on Slack on how to prepare this for a rebase and merge.

**Comment by jasondkiesling** [comment]
This still needs to be rebased per the directions in Slack.

**Comment by samuel-zuk** [comment]
> This still needs to be rebased per the directions in Slack.

my apologies. thought i did it right but was very, very wrong. should be good now though.

**Comment by jasondkiesling** [comment]
I'm not a rebase expert myself so no worries. Can you try rebasing one more time? I'm hoping that it will fix the issue.

**Pull request merged**